site-root/sass/skeleton/modules/_panels.scss

```diff
@@ -0,0 +1,65 @@
+.panel {
```

Comment:
I guess so. It might not be 100% ready to be useful on other sites but it might be a nice thing to have in Skeleton.